src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -114,18 +126,237 @@ class SystemRequestTest
                             am::commands::Command::CommandOrigin::ORIGIN_SDL));
   }
 
+  void Run(MessageSharedPtr message) {
```

Comment:
@BSolonenko such code is widely used in multiple command tests. Please move it to upper level of abstraction.